### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by jonvnadelberg**
this is a fix for https://github.com/sigstore/docs/issues/168

#### Summary

Update [index.md](http://index.md/) (Overview landing page) at top where it ways “Verify with Rekor” → workflow should be “Verify with Cosign” so instructions (and that link) need to be updated, with the assumption that people trying to verify an artifact don’t know anything about Sigstore.

#### Release Note

NONE

#### Documentation

Doc only fix.

### CODE REVIEW

**Comment by jonvnadelberg** [comment]
@olivekl For your review.

File: content/en/index.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ It’s free to use for all developers and software providers, with Sigstore’s
 
 * Sign software artifacts with [Cosign](/cosign/overview/) — [Quick start](/cosign/overview/#quick-start)
 * Sign Git commits with [Gitsign](/gitsign/overview/) - [Quick start](/gitsign/overview/#quick-start)
-* Verify entries with [Rekor](/rekor/CLI/#verify-proof-of-entry)
+* Verify entries with [Cosign](/cosign/verify)
 * Use the [policy controller](/policy-controller/overview/) to enforce Kubernetes policies
 * Use the [Fulcio Certificate Authority](/fulcio/overview/)
```
Comment (haydentherapper) [inline content/en/index.md:21]: I'd drop this too, doesn't seem necessary for the initial landing page

File: content/en/index.md [hunk 2]
```diff
@@ -26,7 +26,7 @@ A Sigstore client, such as Cosign, requests a certificate from Fulcio, a code-si
 
 You don’t have to manage signing keys, and Sigstore services never obtain your private key. The public key that a Sigstore client creates gets bound to the issued certificate, and the private key is discarded after a single signing.
 
-After the client signs the artifact, the artifact's digest, signature and certificate are persisted in Rekor, an immutable, append-only transparency ledger, so that signing events can be publicly audited. Identity owners can monitor the log to verify that their identity is being properly used. This also timestamps the signing event, so that the short-lived certificate can be later verified.
+After the client signs the artifact, the artifact's digest, signature and certificate are persisted in Rekor, an immutable, append-only transparency ledger, so that signing events can be publicly audited. Identity owners can monitor the log to verify that their identity is being properly used. This also timestamps the signing event, so that the short-lived certificate can be later verified. Cosign has the ability to verify.
```
Comment (haydentherapper) [inline content/en/index.md:29]: I don't think this sentence is needed, the next paragraph discusses verification.

Comment (jonvnadelberg) [reply content/en/index.md]: OK.   It looks clumsy to me anyway.  Removed.

File: content/en/index.md (hunk 1 above)
Comment (jonvnadelberg) [reply content/en/index.md:21]: ok, I will take it completely out.

**Review by ltagliaferri — APPROVED** [review]

**Comment by olivekl** [comment]
Will you be updating the instructions in a different PR? Currently they jump to cosign/verify skips the install step.

**Comment by olivekl** [comment]
Okay, my main comment remains: if you link to cosign/verify, the user will have skipped over the installation instructions. The link drops you at the command for verification. This skips over important steps in the user journey for a first-time user trying to verify something if they've never used Sigstore before. 

Can you edit the verification instructions to make it clear that the user needs to install Cosign first? Or otherwise adjust the way the instructions are linked, so the path they need to follow is clear?

**Comment by jonvnadelberg** [comment]
If you look at
https://github.com/jonvnadelberg/sigstore-docs/blob/update-index.md/content/en/index.md
you'll see that the instruction list has "install cosign" as the first
thing to do, with a link to the cosign installation page. The verify link
is completely removed Is this not sufficient?


   - Install Cosign - Cosign Installation page
   <https://github.com/jonvnadelberg/sigstore-docs/blob/update-index.md/cosign/installation>
   - Sign software artifacts with Cosign
   <https://github.com/jonvnadelberg/sigstore-docs/blob/update-index.md/cosign/overview>
    — Quick start
   <https://github.com/jonvnadelberg/sigstore-docs/blob/update-index.md/cosign/overview/#quick-start>
   - Sign Git commits with Gitsign
   <https://github.com/jonvnadelberg/sigstore-docs/blob/update-index.md/gitsign/overview>
    - Quick start
   <https://github.com/jonvnadelberg/sigstore-docs/blob/update-index.md/gitsign/overview/#quick-start>
   - Use the policy controller
   <https://github.com/jonvnadelberg/sigstore-docs/blob/update-index.md/policy-controller/overview>
to
   enforce Kubernetes policies

**Pull request merged**